packages/contracts/contracts/L1/ValidatorPool.sol

```diff
@@ -423,8 +446,10 @@ contract ValidatorPool is ReentrancyGuardUpgradeable, Semver {
         uint256 balance = balances[_validator] + _amount;
 
         if (balance >= REQUIRED_BOND_AMOUNT && !isValidator(_validator)) {
-            validatorIndexes[_validator] = validators.length;
-            validators.push(_validator);
+            if (_validator != SECURITY_COUNCIL) {
```

Comment:
Why don't we add this check to `isValidator` function?
It's optional.

Reply:
When deposit, it can only add a validator if the sender is not already a validator, and it use `!isValidator()` to check this. 
If we add a Security council check in `isValidator()`, the function will returns `false`, and it will be added as a validator.

Reply:
Ah, you're right.